addons/mail/static/src/messaging/entity/thread/thread.js

```diff
@@ -536,6 +801,27 @@ function ThreadFactory({ Entity }) {
             return `${this.constructor.entityName}_${threadModel}_${id}`;
         }
 
+        /**
+         * @private
+         * @param {Object} param0
+         * @param {boolean} param0.isTyping
+         */
+        async _notifyCurrentPartnerTypingStatus({ isTyping }) {
+            if (
+                this._forceNotifyNextCurrentPartnerTypingStatus ||
+                isTyping !== this._currentPartnerLastNotifiedIsTyping
+            ) {
+                await this.async(() => this.env.rpc({
+                    model: 'mail.channel',
+                    method: 'notify_typing',
+                    args: [this.id],
+                    kwargs: { is_typing: isTyping },
+                }, { shadow: true }));
+            }
+            this._forceNotifyNextCurrentPartnerTypingStatus = false;
+            this._currentPartnerLastNotifiedIsTyping = isTyping;
+        }
```

Comment:
shouldn't `_currentPartnerLongTypingTimer` be reset when sending any other status?